config/locales/en.yml

```diff
@@ -216,6 +216,8 @@ en:
     project_categories_team: 'Team'
     project_categories_independent: 'Independent'
   projects:
+    input_name: 'Name*'
+    input_category: 'Category*'
```

Comment:
Not actually a good practice to write asterisk used for required in internationalized text.

Reply:
will do in next PR